packages/commonwealth/client/scripts/views/components/component_kit/cw_icons/cw_custom_icon.tsx

```diff
@@ -9,12 +9,12 @@ import { CustomIconAttrs } from './types';
 import { ComponentType } from '../types';
 
 export class CWCustomIcon implements m.ClassComponent<CustomIconAttrs> {
-  view(vnode) {
+  view(vnode: m.Vnode<CustomIconAttrs>) {
     const {
       componentType = ComponentType.CustomIcon,
       iconName,
       iconSize = 'medium',
-      ...domAttrs
+      ...otherAttrs
```

Comment:
why did this change?

Reply:
`dom` has too much baggage as a word here. I just want to convey that overloads might get passed through to the DOM.

Reply:
Maybe it doesn't matter. 🤷

Reply:
all good